src/chrome/content/rules/Google-mismatches.xml

```diff
@@ -1,27 +0,0 @@
-<!--
-	For other Google coverage, see GoogleServices.xml.
-	
-	Problematic domains:
-
-		- (www.)apture.com	(works, mismatched, CN: *.google.com)
-
--->
-<ruleset name="Google (mismatches)" default_off="mismatched">
-
-	<!--	Akamai	-->
-	<target host="js.admeld.com"/>
```

Comment:
`admeld.com` and `www.admeld.com` can be secured.

Reply:
https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/Admeld.xml